packages/ui-library/src/components/svgicon/index.tsx

```diff
@@ -1,4 +1,16 @@
+import React from 'react';
 import type { SvgIconProps } from '@mui/material/SvgIcon';
 import SvgIcon from '@mui/material/SvgIcon';
 
-export { SvgIcon, type SvgIconProps };
+interface CustomIconProps extends SvgIconProps {
+  viewBox: string;
+  size?: string;
+}
+
+const CustomSvgIcon = ({ viewBox, size, children, ...props }: CustomIconProps) => (
```

Comment:
why did we need this change?

Reply:
For cases like the `ContentCopyIcon`, we are migrating to a custom svg icon. The native Mui <SvgIcon /> wrapper, wasn't supporting custom props such as `onClick`. This structure allows to send any kind of prop and it should be properly included in the icon.